.github/workflows/android-ci.yml

```
@@ -58,10 +58,21 @@ jobs:
             platforms;android-${{ env.ANDROID_API_LEVEL }}
             build-tools;${{ env.ANDROID_BUILD_TOOLS_VERSION }}
 
+      - name: Decode debug keystore
+        if: secrets.DEBUG_KEYSTORE_BASE64 != ''
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)